lib/tasks/docker.rake

```diff
@@ -7,14 +7,20 @@ namespace :docker do
 
   task :build_base do
     Dir.chdir("#{File.dirname(__FILE__)}/../../docker-images/base") do
-      cmd = 'docker build --no-cache=true -t ryuzee/slidehub-base .'
+      cmd = 'docker build --no-cache=true -t ryuzee/slidehub-base:latest .'
       sh cmd
     end
+
+      cmd = "docker tag ryuzee/slidehub-base:latest ryuzee/slidehub-base:20180203"
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.


<!-- Comment by [SideCI](https://sideci.com) -->